package.json

```diff
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "execa": "^1.0.0",
-    "glob": "^7.0.0",
+    "fast-glob": "^3.3.2",
```

Comment:
Also, I would like to exclude this part from this PR. My change will take care of switching over to `fast-glob`. I'd like to merge your change before mine though to make sure it is backward-compatible with "plain" `glob. Then I'll rebase my change on top of yours to test it with `fast-glob`.

It's also fine if you want to test this with `fast-glob` locally in your checkout. We should just remove this part before merging this on the `master` branch.